syntax = "proto2";
package rare.protos;

import "rare/core/rnn_cell.proto";
import "rare/core/hyperparams.proto";


message BidirectionalRnn {
  optional RnnCell fw_bw_rnn_cell = 1;
  optional Regularizer rnn_regularizer = 2;
  optional int32 num_output_units = 3;
  optional Hyperparams fc_hyperparams = 4;
}
